recipe/meta.yaml

```diff
@@ -21,6 +21,10 @@ requirements:
     - {{ compiler('c') }}                # [not win]
     - vs2017_win-64                      # [win64]
```

Comment:
Does this really need 2017 or is 2015 fine, too? If 2015 works, we can use `{{ compiler('c') }}` on all platforms.

Reply:
Also, this is C-only, right? Then please remove `{{ compiler('cxx') }}`.